.github/infra/cloudbuild-configure-triggers.yaml

```diff
@@ -17,12 +17,17 @@
 
 steps:
 - name: 'google/cloud-sdk:slim'
-  id: Configure build triggers deployment
+  id: Configure build triggers
   entrypoint: /bin/bash
   args:
   - -c
   - |
     set -o errexit
     set -o xtrace
-    gcloud beta deployment-manager deployments create cloud-build-triggers --config .github/infra/build-triggers.yaml || \
-      gcloud beta deployment-manager deployments update cloud-build-triggers --config .github/infra/build-triggers.yaml
+    for f in .github/infra/triggers/*.yaml
+    do
+      trigger_filename=$${f##*/}
+      trigger_name=$${trigger_filename%.*} 
+      gcloud beta builds triggers delete --quiet $${trigger_name} || true
```

Comment:
I'm assuming we have to do a delete because import won't do a replace / overwrite?

I guess there will be a small race condition where triggers won't exist between the two commands.

Reply:
> I'm assuming we have to do a delete because import won't do a replace / overwrite?

Correct.

> I guess there will be a small race condition where triggers won't exist between the two commands.

Yes, unfortunately that is true.